### ISSUE TITLE
[Feature Request] Display in dashboards/statistics all payments received

### ISSUE THREAD

**Issue by xSido** [FOSSBilling/FOSSBilling#595]
As title says, would be nice to have displayed all the payments we received (account balance too)
At the moment, only the payments done for services/when client activated a service, is shown in dashboard/statistics.

We receive the payment notification and we can find the payments on user panel and we can also see there his balance (information is not hidden/lost) but since the client can't withdraw the money he paid, i believe we can consider that our income and therefore should be counted on dashboard/statistics too.

**Comment by Anuril** [comment]
From a legal / accounting standpoint: This is not income. It is not our money, it's the clients Money. (borrowed capital) 

If the client wants his money, in most countries you need to refund them. 

Imho, I would close this issue, but what do @jaapmarcus / @John-S4 think?

**Comment by jaapmarcus** [comment]
It doesn't hurt to show all the payments that have been made. But it should not count as income....

**Comment by Anuril** [comment]
What purpose does it serve? I know of no KPI where that would be relevant at the level that FOSSBilling does accounting. 

Total payments received isn‘t a metric that your stripe / paypal / whatever dashboard would not be better equipped to display. 

Maybe unused customer balance would be a metric that would be relevant, as you could see your revenue from actually selling stuff versus money that is „parked“ with you. 

Ultimately, customers who park money with you use you as a bank. And just like with a bank: if you use this money to work with, and the customer wants it back, you need to be able to give it to them. 

Most Business metrics are a relation of two things - bookings versus billing, cashflow in versus cash flow out, etc. 

Just „how much money came in this week“ isn‘t a „front-and-center“ „has to be on the dashboard“ kind of Metric imho. At least not until we can put it into a relation with „how much refunds did I have to give back this week“
